src/isomorphic/React.js

```diff
@@ -35,20 +35,35 @@ if (__DEV__) {
 }
 
 var __spread = Object.assign;
+var createMixin = function(mixin) {
+  return mixin;
+};
 
 if (__DEV__) {
-  var warned = false;
+  var warnedForSpread = false;
+  var warnedForCreateMixin = false;
   __spread = function() {
     warning(
-      warned,
+      warnedForSpread,
       'React.__spread is deprecated and should not be used. Use ' +
       'Object.assign directly or another helper function with similar ' +
       'semantics. You may be seeing this warning due to your compiler. ' +
       'See https://fb.me/react-spread-deprecation for more details.'
     );
-    warned = true;
+    warnedForSpread = true;
     return Object.assign.apply(null, arguments);
   };
+
+  createMixin = function(mixin) {
+    warning(
+      warnedForCreateMixin,
+      'React.createMixin is deprecated and should not be used. You ' +
+      'can use your mixin directly instead.'
```

Comment:
I considered adding some messaging here about mixins being considered harmful, but I'm not sure if that's overstepping since they're not actually considered deprecated.